WooCommerce/Classes/Notifications/LocalNotification.swift

```diff
@@ -18,51 +19,33 @@ struct LocalNotification {
     /// The scenario for the local notification.
     /// Its raw value is used for the identifier of a local notification and also the event property for analytics.
     enum Scenario {
-        case storeCreationComplete(siteID: Int64)
-        case sixHoursAfterFreeTrialSubscribed(siteID: Int64)
-        case freeTrialSurvey24hAfterFreeTrialSubscribed(siteID: Int64)
-        case threeDaysAfterStillExploring(siteID: Int64)
+        case unknown(siteID: Int64)
 
         var identifier: String {
             switch self {
-            case let .storeCreationComplete(siteID):
-                return Identifier.Prefix.storeCreationComplete + "\(siteID)"
-            case let .sixHoursAfterFreeTrialSubscribed(siteID):
-                return Identifier.Prefix.sixHoursAfterFreeTrialSubscribed + "\(siteID)"
-            case let .freeTrialSurvey24hAfterFreeTrialSubscribed(siteID):
-                return Identifier.Prefix.freeTrialSurvey24hAfterFreeTrialSubscribed + "\(siteID)"
-            case let .threeDaysAfterStillExploring(siteID):
-                return Identifier.Prefix.threeDaysAfterStillExploring + "\(siteID)"
+            case let .unknown(siteID):
+                return "unknown" + "\(siteID)"
             }
         }
 
         enum Identifier {
             enum Prefix {
-                static let storeCreationComplete = "store_creation_complete"
-                static let sixHoursAfterFreeTrialSubscribed = "six_hours_after_free_trial_subscribed"
-                static let freeTrialSurvey24hAfterFreeTrialSubscribed = "free_trial_survey_24h_after_free_trial_subscribed"
-                static let threeDaysAfterStillExploring = "three_days_after_still_exploring"
+                static let unknown = "unknown"
             }
         }
 
-        /// Helper method to remove postfix from notification identifiers if needed.
+        /// Helper method to remove postfix from nitificatio identifiers if needed.
         static func identifierForAnalytics(_ identifier: String) -> String {
-            if identifier.hasPrefix(Identifier.Prefix.storeCreationComplete) {
-                return Identifier.Prefix.storeCreationComplete
-            } else if identifier.hasPrefix(Identifier.Prefix.sixHoursAfterFreeTrialSubscribed) {
-                return Identifier.Prefix.sixHoursAfterFreeTrialSubscribed
-            } else if identifier.hasPrefix(Identifier.Prefix.freeTrialSurvey24hAfterFreeTrialSubscribed) {
-                return Identifier.Prefix.freeTrialSurvey24hAfterFreeTrialSubscribed
-            } else if identifier.hasPrefix(Identifier.Prefix.threeDaysAfterStillExploring) {
-                return Identifier.Prefix.threeDaysAfterStillExploring
+            if identifier.hasPrefix(Identifier.Prefix.unknown) {
+                return Identifier.Prefix.unknown
```

Comment:
Nit: we should not need this prefix check, just returning the identifier should be sufficient.

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/pull/12443/commits/ca91d2429c52d062973c89afb2944f84ef95788f